src/types/issuer.ts

```diff
@@ -0,0 +1,8 @@
+import * as t from "io-ts";
+import { string } from "io-ts";
+
+export const Issuer = t.interface({
+  _: string
```

Comment:
why?

Reply:
This definition comes from the [io-backoffice code](https://github.com/teamdigitale/io-backend/blob/master/src/types/issuer.ts#L9). I'll move it inline as you suggested in another comment, but the property name is correct, I inspected the code during its execution and it is `_`.